src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -86,16 +87,16 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
-    public String getSurnameOcrFieldName(String formType) {
-        return formTypeToSurnameOcrFieldMappings.get(formType);
+    public Optional<List<String>> getSurnameOcrFieldNameList(String formType) {
+        return Optional.ofNullable(formTypeToSurnameOcrFieldMappings.get(formType));
```

Comment:
Looks dodgy. If there's a list, shouldn't we return empty list instead?

Reply:
I prefer optional or null.

Reply:
null is never good. it's the worst that could have happen in programming world. In general `Optional<List<T>>` is bad implementation. Either `Optional` or `List`